llvm/lib/Target/ARM/MCTargetDesc/ARMAsmBackend.cpp

```diff
@@ -446,6 +447,16 @@ unsigned ARMAsmBackend::adjustFixupValue(const MCAssembler &Asm,
                                          const MCSubtargetInfo* STI) const {
   unsigned Kind = Fixup.getKind();
 
+  // For MOVW/MOVT Instructions, the Fixup Value needs to be 16 bit aligned.
+  // If this is not the case, we should reject compilation.
```

Comment:
Second sentence seems redundant especially if you rephrase the first to:
```
// For MOVW/MOVT instructions, the fixup value must already be be 16-bit aligned.
```

Reply:
I have updated the comment as per your suggestion